cocos/spine/skeleton.ts

```diff
@@ -94,7 +94,7 @@ ccenum(AnimationCacheMode);
 
 function setEnumAttr (obj, propName, enumDef) {
     CCClass.Attr.setClassAttr(obj, propName, 'type', 'Enum');
-    CCClass.Attr.setClassAttr(obj, propName, 'enumList', Enum.getList(enumDef));
+    CCClass.Attr.setClassAttr(obj, propName, 'enumType', enumDef);
```

Comment:
It can not remove 'enumList' which be used so many places in Editor

Reply:
Maybe the ‘enumType‘ can be an additional prop name，detail with 'type ( Enum) '.

Reply:
I have fixed all editor references in here: https://github.com/cocos/cocos-editor/pull/1959

Reply:
Update: I've added enumList back